docs/reference/flavors/index.md

```diff
@@ -0,0 +1,50 @@
+# Flavors
+
+Any server instance running in {{extra.brand}} Cloud has a **flavor**,
+which defines the number of virtual CPU cores, the amount of virtual
+RAM, and other performance-related factors.
+
+
+## Naming convention
+
+Flavor names in {{extra.brand}} follow a convention, which can be
+summarized as `X.YcZgb`:
+
+* `X` stands for a lowercase letter identifying the [compute
+  tier](#compute-tiers), with `b` representing the general-purpose
+  tier. It is always followed by a full-stop (`.`).
+* `Y` stands for the number of virtual CPU cores. This number is
+  always followed by the letter `c`.
+* `Z` stands for the allocated amount of virtual RAM, in
+  [gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary)). This
+  number is always followed by the string `gb`.
+
+For example, the flavor named `b.4c32gb` would be used for a
+general-purpose compute instance with 4 cores and 32 GiB RAM.
+
+
+## Compute tiers
+
+{{extra.brand}} Cloud defines the following compute tiers:
+
+* `b`: General purpose. This is the default compute tier. Instances
+  launched with matching flavors use highly available network-attached
+  storage. This makes them flexible to migrate within the
+  {{extra.brand}} Cloud infrastructure, without interruption.
```

Comment:
unless encrypted block storage device type is selected

Reply:
... unless customers [follow these suggestions](https://github.com/citynetwork/docs/blob/main/docs/howto/openstack/cinder/encrypted-volumes.md#block-device-encryption-caveats).

Reply:
I've added a note to cover this.